Ex1/5.2.cup_of_join.py

```diff
@@ -0,0 +1,30 @@
+from functools import reduce
+
+
+def all_list(tpl):
```

Comment:
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing